adoc/MAIN.release-notes.adoc

```diff
@@ -37,6 +37,7 @@ This release supports deployment on:
 
 
 // Release specific files, latest on top!
+include::release-4.5.3.adoc[Release 4.5.4]
```

Comment:
Shouldn't that be `release-4.5.4.adoc` instead ?

Reply:
:facepalm: yes you are right